services/app-web/src/pages/StateSubmission/Contacts/Contacts.test.tsx

```diff
@@ -11,8 +11,35 @@ import { renderWithProviders } from '../../../testHelpers/jestHelpers'
 import { Contacts } from './'
 import userEvent from '@testing-library/user-event'
 import { UnlockedHealthPlanFormDataType } from '../../../common-code/healthPlanFormDataType'
+import * as useRouteParams from '../../../hooks/useRouteParams'
+import * as useHealthPlanPackageForm from '../../../hooks/useHealthPlanPackageForm'
 
 describe('Contacts', () => {
+    const mockUpdateDraftFn = jest.fn()
+    beforeEach(() => {
+        jest.spyOn(
```

Comment:
I don't love that we're using spyOn to make these tests work. If we are truly going to be able to delete this code once we have things fully switched over then I'm ok with this as a temporary step, but I really think we should be able to write our tests without having to intercept these kinds of functions in the end. 

Like by mocking the update mutation at the apollo mocks layer instead of intercepting the function call.

Reply:
Yes this is temporary.  We are going to delete all these files tests and the approach to requests will have to be reimplemented for each page anyway with requests that return data directly. @macrael I hope you can be part of that work once API stuff is in better place and we land linked rates. It's a significant lift